learn/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -1,3 +1,93 @@
 # Auth Kit
 
-The Auth Kit uses the [auth-provider](https://github.com/safe-global/account-abstraction-sdk/blob/dev/packages/auth-provider/README.md) to allow users to obtain an Ethereum address using their email or social media account.
+This library provides a way to authenticate blockchain accounts using email addresses, social accounts or traditional web3 wallets (ex. Metamask). When using web2 methods as your email or social account, a derived Ethereum address will be generated.
+
+## Quickstart
+
+### Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-nodejs-and-npm)
+- For using Web3Auth as the authentication provider, you need to have a [Web3Auth account](https://web3auth.io) so sign up if you don't have one already.
+
+### Install dependencies
+
+```bash
+npm install @safe-global/auth-kit
```

Comment:
```suggestion
```

Reply:
Why delete the npm way?

Reply:
It's redundant. Most users already know that `yarn add` can be substituted with `npm install`. To be more explicit, we can add it as a comment in the docs. 

Having both creates more work for the developers. They can't copy-paste the code and run it easily. They'll have to remove the command that's not relevant to them.